modules/renter/downloadchunk.go

```diff
@@ -72,6 +73,10 @@ type unfinishedDownloadChunk struct {
 	// The download object, mostly to update download progress.
 	download *download
 	mu       sync.Mutex
+
+	// Caching related fields
+	chunkCache map[string][]byte
+	cmu        *sync.Mutex
```

Comment:
need to name this cacheMu, cmu isn't sufficiently readable if you haven't memorized the struct.